docs/snakefiles/deployment.rst

```diff
@@ -262,6 +262,30 @@ Conda deployment also works well for offline or air-gapped environments. Running
 
 .. _singularity:
 
+
+-------------------------
+Providing post-deployment scripts
+-------------------------
+
+With Snakemake 9.14 post-deployment shell-scripts can be provided to perform additional adjustments of a conda environment.
+This might be helpful in case a conda package is missing components or requires further configuration for execution.
+post-deployment scripts must be placed next to their corresponding environment-file and require the suffix ``.post-deploy.sh``.
+
+.. code-block:: python
+
+    rule NAME:
+        input:
+            "seqs.fastq"
+        output:
+            "results.tsv"
+        conda:
+            "envs/interproscan.yaml"
+            "envs/interproscan.post-deploy.sh
```

Comment:
```suggestion
```
The plan was to have them in the same folder, but there is no need to mention them in the conda env definition.

Reply:
True. My example is wrong. I'll update the documentation.